TESTING.md

```diff
@@ -5,64 +5,50 @@
 - [Integration tests](#integration-tests)
 - [Rake Tasks](#rake-tasks)
 
-## Syntax and style
+## Requirements
 
-### Requirements:
+- [ChefDK](https://downloads.chef.io/chefdk)
+- [Vagrant](https://www.vagrantup.com/)
+- [Packer](https://www.packer.io/)
+- A supported macOS hypervisor:
+  - [Parallels](https://www.parallels.com/landingpage/pd/general/)
+  - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+  - [VMWare Fusion](https://www.vmware.com/products/fusion.html)
 
-- `cookstyle` and `foodcritic` (both are shipped with the [ChefDK](https://downloads.chef.io/chefdk))
+## Syntax and style
 
-### Running the tests
+- `cookstyle` and `foodcritic`
 
 Syntax testing is pretty straight forward. At the root of the cookbook, run:
 
 ```shell
-$ cookstyle
-```
-
-and
-
-```
-$ foodcritic .
+cookstyle
+foodcritic .
 ```
 
 ## Unit tests
 
-### Requirements:
-
-- [RSpec](http://rspec.info/)
-- [ChefDK](https://downloads.chef.io/chefdk) (optional)
-
 For unit tests, we focus on testing the library files, which are written
 in pure Ruby and tested with RSpec. The library files contain most of the core
 business logic for each of the custom resources and are used as either mixins
 for the custom resources or contain classes that act as helpers, except with
 explicit namespacing. Some libary files are heavily unit tested, others are definitely
 missing much-needed unit test coverage.
 
-### Running the tests
-
 Clone this repo and in the root of the cookbook, run:
 
 ```shell
-$ rspec spec
+rspec spec
```

Comment:
Do we use `chef exec spec` since we're running ChefSpec?

Reply:
This just assumes you are using `chef shell-init bash` to prevent needing all the `chef exec` prefixes. I will add a note about that.